google/earthenterprise
# Bundle Qt4 and build it in Scons

Bundles Qt4 (so we have it for all platforms) and builds it in scons. Qt4 shared objects are also installed for runtime.

To test (on El6, El7, EL8, and Ubuntu):

1. Build and install
2. Verify that fusion works, and that Qt shared object files are installed under `/opt/google/lib`
